cimi-resources/test/com/sixsq/slipstream/ssclj/resources/spec/storage_bucket_test.cljc

```diff
@@ -0,0 +1,70 @@
+(ns com.sixsq.slipstream.ssclj.resources.spec.storage-bucket-test
+  (:require
+    [clojure.test :refer [deftest]]
+    [com.sixsq.slipstream.ssclj.resources.spec.spec-test-utils :as stu]
+    [com.sixsq.slipstream.ssclj.resources.spec.storage-bucket :as bucky]
+    [com.sixsq.slipstream.ssclj.resources.storage-bucket :as bucky-resource]))
+
+
+(def valid-acl {:owner {:principal "ADMIN",
+                        :type      "ROLE"},
+                :rules [{:principal "realm:accounting_manager",
+                         :type      "ROLE",
+                         :right     "VIEW"},
+                        {:principal "test",
+                         :type      "USER",
+                         :right     "VIEW"},
+                        {:principal "cern:cern",
+                         :type      "ROLE",
+                         :right     "VIEW"},
+                        {:principal "cern:my-accounting-group",
+                         :type      "ROLE",
+                         :right     "VIEW"}]})
+
+
+(def timestamp "1964-08-25T10:00:00.0Z")
+
+
+(def bucky-sample {:id           (str bucky-resource/resource-url "/uuid")
+                :resourceURI  bucky-resource/resource-uri
+                :created      timestamp
+                :updated      timestamp
+                :acl          valid-acl
+                :name         "short name"
+                :description  "short description",
+                :properties   {:a "one", :b "two"}
+                :bucketName   "aaa-bbb-111"
+                :usage        123.456
+                :connector    {:href "connector/0123-4567-8912"}
+                :credentials  [{:href  "connector/0123-4567-8912",
+                                :roles ["realm:cern", "realm:my-accounting-group"]
+                                :users ["long-user-id-1", "long-user-id-2"]}]
```

Comment:
You can leave out the commas from vectors and lists.

Reply:
fixed